lib/board.rb

```diff
@@ -22,11 +22,18 @@ def initialize
               }
   end
   # looping through - nested loop for expansion
+  # should the methods below be a placement class?
 
   def valid_coordinate?(coordinate)
     cells.keys.include?(coordinate)
   end
 
+  def all_coordinates_valid?(placement_coordinates) 
+    placement_coordinates.all? do |coordinate|
+      valid_coordinate?(coordinate)
+    end
+  end
+
   def all_same?(array)
```

Comment:
I feel like we're going to need to do some major reordering of methods to make this readable, with some of these tiny methods below. These might actually belong in another class? Could belong to the placement.

Reply:
Yes! Absolutely agreed!

Reply:
Tabling this for next round.